patches/net/minecraft/world/level/chunk/ChunkAccess.java.patch

```diff
@@ -111,5 +111,5 @@
 +        return attachmentHolder;
 +    }
 +    @Nullable
-+    public net.minecraft.world.level.LevelAccessor getWorldForge() { return null; }
++    public net.minecraft.world.level.Level getWorldForge() { return null; }
```

Comment:
If the return type is changed anyway, then we can just rename this to `getLevel()` and remove the delegating override in `LevelChunk` since there is no obfuscation anymore that would cause the vanilla method in `LevelChunk` to not override this one in a production environment.